src/main/scala/strawman/collection/Seq.scala

```diff
@@ -72,6 +72,14 @@ trait IndexedSeqOps[+A, +CC[X] <: IndexedSeq[X], +C] extends Any with SeqOps[A,
     def length: Int = self.length
     def apply(i: Int): A = self(i)
   }
+
+  /** A collection containing the last `n` elements of this collection. */
+  override def takeRight(n: Int): C = fromSpecificIterable(view.takeRight(n))
```

Comment:
My proposed change won't work since it leads to infinite loops via `View` and `IterableOps` calling each other. The call in `IterableOps` need to call a method that is not delegating back to the implementation inherited from `IterableOps` (which the methods in `View` do, but the ones in the operation specific `View` subclass do not, since they call the same operation on the underlying collection's iterator instead).